pom.xml

```diff
@@ -1,433 +1,472 @@
+				<artifactId>tycho-p2-plugin</artifactId>
+			</plugin>
+
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<version>${tychoVersion}</version>
+				<configuration>
+					<resolver>p2</resolver>
+					<pomDependencies>consider</pomDependencies>
+					<target>
+						<artifact>
+							<groupId>org.jboss.tools.integration-stack</groupId>
+							<artifactId>target-platform</artifactId>
+							<version>${jbtis.version}</version>
+							<classifier>${jbtis.classifier}</classifier>
+						</artifact>
+					</target>
+					<ignoreTychoRepositories>false</ignoreTychoRepositories>
+					<environments>
+						<environment>
+							<os>win32</os>
+							<ws>win32</ws>
+							<arch>x86</arch>
+						</environment>
+						<environment>
+							<os>win32</os>
+							<ws>win32</ws>
+							<arch>x86_64</arch>
+						</environment>
+						<environment>
+							<os>linux</os>
+							<ws>gtk</ws>
+							<arch>x86_64</arch>
+						</environment>
+						<environment>
+							<os>linux</os>
+							<ws>gtk</ws>
+							<arch>x86</arch>
+						</environment>
+						<environment>
+							<os>macosx</os>
+							<ws>cocoa</ws>
+							<arch>x86_64</arch>
+						</environment>
+					</environments>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.19.1</version>
+				<executions>
+					<execution>
+						<id>test</id>
+						<phase>test</phase>
+						<configuration>
+							<systemProperties>
+								<osgi.nls.warnings>ignore</osgi.nls.warnings>
+							</systemProperties>
+						</configuration>
+						<goals>
+							<goal>test</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<version>${tychoVersion}</version>
+				<configuration>
+					<useUIThread>true</useUIThread>
+					<useUIHarness>true</useUIHarness>
+					<includes>
+						<include>**/*IT.class</include>
+					</includes>
+					<argLine>${tycho.testArgLine} -XX:+HeapDumpOnOutOfMemoryError
+						-Dusage_reporting_enabled=false
+						-Dorg.jboss.tools.vpe.loadxulrunner=false</argLine>
+					<osgiDataDirectory>${project.basedir}/target/work sp@cé</osgiDataDirectory>
+				</configuration>
+			</plugin>
+		</plugins>
+	</build>
+
+	<dependencies>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+
+	<profiles>
+		<profile>
+			<id>osx</id>
+			<activation>
+				<property>
+					<name>java.vendor.url</name>
+					<value>http://www.apple.com/</value>
+				</property>
+			</activation>
+			<properties>
+				<!-- THE FOLLOWING LINE MUST NOT BE BROKEN BY AUTOFORMATTING -->
+				<systemProperties combine.children="append"> -Dosgi.arch=x86
```

Comment:
this should stay on the same line. I think that on recent version of Tycho, w can provide several tags to allow writing on different lines